docs/content/latest/deploy/checklist.md

```diff
@@ -139,6 +139,18 @@ In practice, the clock drift would have to be orders of magnitude higher in orde
 
 {{< /note >}}
 
+## DNS caching
+
+If you are using name servers instead of static ips you may end up doing aggressive DNS lookups.
+
+If the DNS query load is large enough to be a problem or the RTT to the external DNS server is long enough to be a problem, you 
+can set high enough TTL on your DNS records.
```

Comment:
```suggestion
If the DNS query load is large enough to be a problem, or the RTT to the external DNS server is long enough to be a problem, you can use higher TTL valuea on your DNS records.
```

Reply:
The comments "large enough to be a problem" and "long enough to be a problem" seem too vague to me. Can you provide a better explanation for users?